code/extensions/WorkflowEmbargoExpiryExtension.php

```diff
@@ -358,4 +359,26 @@ public function setIsWorkflowInEffect() {
 	public function getIsWorkflowInEffect() {
 		return $this->isWorkflowInEffect;
 	}
+
+    /**
+     * Add edit check for when publishing has been scheduled and if any workflow definitions want the item to be disabled.
+     */
+    public function canEdit($member) {
+        $disabled = false;
+        $definitions = $this->workflowService->getDefinitionsFor($this->owner);
```

Comment:
Can you add a check for where `this->owner->PublishOnDate` is actually defined before querying the workflow definitions?